misc/discoverynode/README.md

```diff
@@ -8,5 +8,13 @@
 `~/venv/bin/python3 -m pytest tests/`
 
 ## Integration test (no MQTT)
-`sudo testing/integration/integration_test.sh`
+
+```
+sudo testing/integration/integration_test.sh
+
+sudo docker ps -a | grep "discoverynode-test-" | awk '{print $1}' | sudo xargs docker stop
```

Comment:
just trying to understand this -- it's stopping a different container than it starts?